src/learn/augur-to-auspice.rst

```diff
@@ -574,24 +574,51 @@ following example may illustrate what's possible:
 gray in the tree and the “unknown” values will not be present in the color legend.
 
-.. _auspice-config-how-branch-mutations-are-exported:
+.. _auspice-config-branch-mutations-and-diversity-panel:
 
-How branch mutations are exported
----------------------------------
+Branch Mutations and Diversity (Entropy) Panel
+----------------------------------------------
 
-Nucleotide and amino-acid mutations are provided via node-data JSONs
-under the keys ``muts`` (list) and ``aa_muts`` (dictionary), along with
-``annotations`` (dictionary). These are typically produced by
-``augur ancestral`` or ``augur translate`` and so aren't detailed here.
-These metadata are exported as ``node.branch_attrs.mutations`` and
-``meta.genome_annotations``.
+The branch mutations and the diversity (entropy) panel are enabled by the same
+nucleotide mutations, amino acid mutations, and genome annotations provided
+via the dataset JSON. The genome annotations are **required** for the
+:ref:`diversity panel in Auspice <auspice-component-diversity-panel>`.
 
 If these metadata are available, then a special coloring is created in
 the exported dataset JSON: ``{"key": "gt", "title": "Genotype", "type": "categorical"}``
 which is used by Auspice to allow coloring by genotype.
 You can also define this in the auspice-config JSON if you wish to use a
-different title. The presence of these metadata will also enable the
-:ref:`diversity panel in Auspice <auspice-component-diversity-panel>`.
+different title.
+
+Nucleotide Mutations
+~~~~~~~~~~~~~~~~~~~~
+The nucleotide mutations are assigned to internal nodes via ``augur ancestral``.
+The mutations are output to a node-data JSON as ``nodes.<node_name>.muts``.
+The node-data JSON is passed to ``augur export v2`` via the ``--node-data`` option
+and the nucleotide mutations are exported as ``node.branch_attrs.mutations.nuc``.
+
+See :doc:`augur ancestral <augur:usage/cli/ancestral>` for more details on
+how the nucleotide mutations are assigned.
+
+Amino Acid Mutations
+~~~~~~~~~~~~~~~~~~~~
+The node-data JSON output from ``augur ancestral`` includes the nucleotide sequences
+for internal nodes as ``nodes.<node_name>.sequence``, which are passed to ``augur translate``
+to translate and generate the amino acid mutations. The mutations are output to
+a node-data JSON as ``nodes.<node_name>.aa_muts.<gene_name>``.
+The node-data JSON is passed to ``augur export v2`` via the ``--node-data`` option
+and the amino acid mutations are exported as ``node.branch_attrs.mutations.<gene_name>``.
+
+See :doc:`augur translate <augur:usage/cli/translate>` for more details on
+how the amino acid mutations are generated.
+
+Genome Annotations
+~~~~~~~~~~~~~~~~~~
+The genome annotations are also generated by ``augur translate``.
+The features of the reference sequence, provided via ``--reference-sequence``,
+are formatted according to the `annotations schema <https://github.com/nextstrain/augur/blob/master/augur/data/schema-annotations.json>`_
```

Comment:
I was _going_ to say, "Let's link this to the canonical URL/id of the schema for future-proofing?" and suggest a URL under https://nextstrain.org/schemas/… but then realized this is the sole schema we don't publish that way, because it's only really for a document fragment.

We should indeed revisit this partial schema, as noted in the commit which introduced it (https://github.com/nextstrain/augur/commit/c1b2cb592fd94abba418d32a40b9e9904a165ed2). I think a generic node-data schema is easily doable if we want to do that, or we can define separate schemas for each and detect the node data type thru validation.